src/test/rustdoc/tuple_struct_fields.rs

```diff
@@ -0,0 +1,11 @@
+// @has tuple_struct_fields/struct.Tooople.html
+// @has - //span '0: usize'
```

Comment:
Could you write more complete paths please? It's much easier when an unwanted change occur to find out where it's from in such cases.

Reply:
As i stated in an earlier comment, i tried that, but the test harness kept erroring and i could not figure out why, which is why i removed it

Reply:
I can help with that! :) If it's the same as a normal field, the path should look like this:

```
//section[@id="main"]/span[@id="structfield.0"] '0: usize'
```

Also, please add tests that "Tuple fields" header is there and that the fields are in the sidebar too.